app/util/data_preparation/crowd_prepare_data.py

```diff
@@ -29,7 +29,8 @@ def __get_users(crowd_api, count):
         print('All performance test users were successfully created')
         return cur_perf_users
     else:
-        raise Exception(f'Your Atlassian Crowd instance does not have enough users {len(cur_perf_users)} out of {count}')
+        raise SystemExit(f'Your Atlassian Crowd instance does not have enough '
+                         f'users {len(cur_perf_users)} out of {count}')
```

Comment:
Your Atlassian Crowd instance does not have enough users. Current users count {len(cur_perf_users)} out of {count}.